### PR TITLE
NEOS-824: updating styles and layout of login page

### PR DESCRIPTION
**Pull request by evisdrenova**

### CODE REVIEW

File: src/login/Template.tsx [hunk 1]
```diff
@@ -132,12 +77,24 @@ export default function Template(props: TemplateProps<KcContext, I18n>) {
                     {msg("requiredFields")}
                   </span>
                 </div>
-                <div className="col-md-10">
-                  <h1 id="kc-page-title">{headerNode}</h1>
+                <div className="headerLogo">
+                  <img
+                    src={`${url.resourcesPath}/img/logo_and_text_light_mode.svg`}
```
Comment (nickzelei) [inline src/login/Template.tsx:82]: all of these images are broken for me locally. how can I get them to render properly?

File: package.json [hunk 2]
```diff
@@ -43,6 +43,7 @@
     "eslint-plugin-react-refresh": "^0.4.5",
     "eslint-plugin-storybook": "^0.6.15",
     "storybook": "^7.6.14",
+    "tailwindcss": "^3.4.1",
```
Comment (nickzelei) [inline package.json:46]: do we need this?

**Review by nickzelei — COMMENTED** [review]
overall I think the two pages look good, but I need to figure out how to get the images to load.
Also, it looks like the header title was removed. My only concern around that is accessiblity. Nowhere on the page will it say "Neosync" as it is all just images.

File: package.json (hunk 2 above)
Comment (evisdrenova) [reply package.json:46]: nope - removed it.

File: src/login/Template.tsx (hunk 1 above)
Comment (evisdrenova) [reply src/login/Template.tsx:82]: not sure why they wouldn't load locally but just to be safe I updated the images to pull from S3

**Review by nickzelei — APPROVED** [review]

**Pull request merged**